### ISSUE TITLE
Respec warnings related to missing <dfn> and Object error

### ISSUE THREAD

**Issue by vivienlacourba** [w3c/webrtc-pc#978]
I noticed some respec warnings while running:
`cd webrtc-pc && git checkout master && git pull`
`make clean && make update && make check`

```
...
node support/respec/tools/respec2html.js -e --src file://`pwd`/webrtc.html --out build/output.html
🚨 ReSpec warn: No <dfn> for rtcconfiguration.
🚨 ReSpec warn: No <dfn> for rtcicecredentialtype.
🚨 ReSpec warn: No <dfn> for rtciceserver.
🚨 ReSpec warn: No <dfn> for rtcicetransportpolicy.
🚨 ReSpec warn: No <dfn> for rtcbundlepolicy.
🚨 ReSpec warn: No <dfn> for rtcrtcpmuxpolicy.
🚨 ReSpec warn: No <dfn> for rtcofferansweroptions.
🚨 ReSpec warn: No <dfn> for rtcofferoptions.
🚨 ReSpec warn: No <dfn> for rtcansweroptions.
🚨 ReSpec warn: No <dfn> for addicecandidate!overload-1 in rtcpeerconnection.
🚨 ReSpec warn: No <dfn> for rtcsignalingstate.
🚨 ReSpec warn: No <dfn> for rtcicegatheringstate.
🚨 ReSpec warn: No <dfn> for rtcpeerconnectionstate.
🚨 ReSpec warn: No <dfn> for rtciceconnectionstate.
🚨 ReSpec warn: No <dfn> for rtcpeerconnectionerrorcallback.
🚨 ReSpec warn: No <dfn> for rtcsessiondescriptioncallback.
🚨 ReSpec warn: No <dfn> for rtcstatscallback.
🚨 ReSpec warn: No <dfn> for rtcsdptype.
😱 ReSpec error: [object Object]
```

**Comment by aboba** [comment]
@vivienlacourba The warnings appear to have been transient (the ORTC API spec at draft.ortc.org saw them as well, but now they are gone).

**Comment by alvestrand** [comment]
@aboba should make sure there's a link to the PR when adding "PR exists"...

**Comment by dontcallmedom** [comment]
See #989

**Comment by alvestrand** [comment]
Mostly fixed by #989 - objects still missing.

**Comment by alvestrand** [comment]
Hopefully fixed with #995
